share/spice/quandl/home_values/content.handlebars

```diff
@@ -0,0 +1,20 @@
+
+<div id="quandl">
+    <h1 class="quandl-header">
```

Comment:
Are there pre-defined styles for zci__header, or should I simply rename the .quandl-header class in my css to zci__header?  Are there DDG docs on css?